community/cypher/runtime-util/src/main/scala/org/neo4j/cypher/internal/runtime/planDescription/InternalPlanDescription.scala

```diff
@@ -276,7 +281,7 @@ final case class PlanDescriptionImpl(id: LogicalPlanId,
 
   override def toString = {
     val version = arguments.collectFirst {
-      case Version(v) => s"Compiler $v$NL"
+      case RuntimeVersion(v) => s"Compiler $v$NL"
```

Comment:
I wonder if `Compiler` should be changed to `Runtime`. The word 'compiler' before meant both planner and runtime, but now we have a clearer separation between planner and runtime.